src/xrpld/app/misc/WasmHostFuncImpl.cpp

```diff
@@ -733,6 +801,26 @@ WasmHostFunctionsImpl::traceNum(std::string_view const& msg, int64_t data)
     return msg.size() + sizeof(data);
 }
 
+Expected<int32_t, HostFunctionError>
+WasmHostFunctionsImpl::traceAccount(
+    std::string_view const& msg,
+    AccountID const& account)
+{
+#ifdef DEBUG_OUTPUT
+    auto j = getJournal().error();
+#else
+    auto j = getJournal().trace();
+#endif
+    if (!account)
+        return Unexpected(HostFunctionError::INVALID_ACCOUNT);
```

Comment:
No error trace?

Reply:
There's no error trace in any other host function...?